app/assets/javascripts/rat_adoptions.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
as you don't have any javascript in here yet, how would you feel about removing this file?